docs/user-manual/en/amqp-broker-connections.md

```diff
@@ -32,121 +32,145 @@ To define an AMQP broker connection, add an `<amqp-connection>` element within t
+* `queue-creation`: Specifies whether a queue- or address-creation event is sent. The default value is `true`.
+
+An example of a mirror configuration is shown below:
 ```xml
 <broker-connections>
-    <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker">
-        <receiver queue-name="remoteQueueA"/>
-        <sender queue-name="localQueueB"/>
-    </amqp-connection>
+  <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-mirror">
+    <mirror  queue-removal="true" queue-creation="true" message-acknowledgements="true" source-mirror-address="myLocalSNFMirrorQueue"/>
+  </amqp-connection>
 </broker-connections>
+```
 
-<addresses>
-     <address name="remotequeues.A">
-        <anycast>
-           <queue name="remoteQueueA"/>
-        </anycast>
-     </address>
-     <address name="queues.B">
-        <anycast>
-           <queue name="localQueueB"/>
-        </anycast>
-     </address>
-</addresses>
+
+### Store and Forward Queue
+
+Mirror events are always stored on a local queue prefixed as "$ACTIVEMQ_ARTEMIS_MIRROR_" concatenated with broker connection's name.
+
+So, in the following configuration mirror events will be stored on a queue named "$ACTIVEMQ_ARTEMIS_MIRROR_my-mirror".
+
+```xml
+<broker-connection>
+  <amqp-connection uri="tcp://brokerB:5672" name="brokerB">
+    <mirror/>
+  </amqp-connection>
+</broker-connection>
 ```
-*Important:* You can match a receiver only to a local queue that already exists. Therefore, if you are using receivers, make sure that you pre-create the queue locally. Otherwise, the broker cannot match the remote queues and addresses.
 
-*Important:* Do not create a sender and a receiver to the same destination. This creates an infinite loop of sends and receives.
+These messages are then transferred to brokerB:5672. A link with the address name $ACTIVEMQ_ARTEMIS_MIRROR_brokerB will be created towards brokerB. If there is a security manager configured, security roles must be provided to the user on the broker connection.
+
+Notice the queue $ACTIVEMQ_ARTEMIS_MIRROR_brokerB will not physically exist on brokerB and there won't be a way to find it on the administrator console. The broker will treat these messages accordingly as mirror events and will execute the appropriate operations at the target broker.
+
+### Pre Existing Messages
+The brokers will only send messages from the point in time the mirror was configured. Previously existing messages will not be forwarded to other brokers.
+
+## Dual Mirror (Disaster and Recovery option)
+
+ActiveMQ Artemis supports automatic failback mirroring. Every sent message and every acknowledgement is asynchronously replicated to the mirrored broker.
+
+On the following diagram, there will be two servers called DataCenter1, and DataCenter2. In order to have a dual mirror configuration, it is necessary is to add the mirror broker connection on each broker.xml:
+
+![Broker Connection DR](images/broker-connection-DR.jpg)
```

Comment:
I would personally add two lines with a single arrow heads at opposoding ends, rather than 1 line with arrow heads at both ends, its more descriptive of what occurs. Again I would remove the config from the diagrams.